### DISCUSSION TITLE
Petlibro PLAF103 being detected as incorrect device

### DISCUSSION THREAD

**Discussion by mjakeseaton** [make-all/tuya-local#1616] [Q&A]
I have the Petlibro PLAF103 Wifi Feeder. When I add the Device ID, local_key and protocol version 3.4, tuya-local is detecting the device as a catit_pixi_6meal_feeder, which does not have the correct entities.

Any idea how to get tuya-local to detect the feeder correctly?

**Comment by mjakeseaton** [comment]
Followed the guidance in this [discussion](https://github.com/make-all/tuya-local/discussions/1544) to manually modify the `core.config_entries` file.

1. Edited the `title` to `Petlibro PLAF103 Wifi Feeder` (not necessary)
2. Edited the `type` to `petlibro_PLAF103_feeder`
3. Saved and rebooted
4. Removed any device entities that were specific to the `catit_pixi_6meal_feeder`

**Reply by make-all** [reply]
Manually tinkering with the device registry like this is not recommended, as it will get out of sync with the entity registry.
If you send the log message that contains the dps list when the misdetection happens, I would be able to fix the detection.

**Reply by mjakeseaton** [reply]
@make-all I'll be happy to provide those logs. Would you mind pointing me to where I could pull those? I checked my HA Core logs, and the only reference to Tuya Local is this:

```
Logger: homeassistant.util.async_
Source: util/async_.py:141
First occurred: February 5, 2024 at 9:54:46 PM (2 occurrences)
Last logged: February 5, 2024 at 9:54:50 PM

Detected blocking call to sleep inside the event loop by custom integration 'tuya_local' at custom_components/tuya_local/device.py, line 77: self._api = tinytuya.Device(dev_id, address, local_key), please create a bug report at https://github.com/make-all/tuya-local/issues
```
